src/main/java/teammates/ui/controller/InstructorFeedbackRemindParticularStudentsPageData.java

```diff
@@ -1,14 +1,43 @@
 package teammates.ui.controller;
 
+import java.util.List;
+import java.util.ArrayList;
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.FeedbackSessionResponseStatus;
+import teammates.ui.template.RemindParticularStudentsCheckboxEmailNamePair;
 
 public class InstructorFeedbackRemindParticularStudentsPageData extends PageData {
     public FeedbackSessionResponseStatus responseStatus;
     public String courseId;
     public String fsName;
+    private List<RemindParticularStudentsCheckboxEmailNamePair> emailNamePairs;
```

Comment:
Try to keep attributes `private`, pass them in via the constructor or `init()`.
It's fine just putting the variables to be passed in within the constructor's signature since it's probably just a bunch of `this.* = *`.